packages/angular_devkit/build_angular/src/karma/index.ts

```diff
@@ -71,6 +71,13 @@ export class KarmaBuilder implements Builder<KarmaBuilderSchema> {
           karmaOptions.browsers = options.browsers.split(',');
         }
 
+        if (options.reporters) {
+          // Split along commas to make it more natural, and remove empty strings.
+          karmaOptions.reporters = options.reporters
+            .reduce<string[]>((acc, curr) => acc.concat(...curr.split(/,/)), [])
```

Comment:
Is the spread operator really needed here?

Reply:
Removed.